### PR TITLE
minor fix

### PR DESCRIPTION
**Pull request by MOON-CLJ**
cc @everpcpc

### CODE REVIEW

File: include/rapidjson/itoa.h [hunk 1]
```diff
@@ -294,12 +295,13 @@ inline char* u64toa(uint64_t value, char* buffer) {
...
+    uint64_t u = static_cast<uint64_t>(value);
     if (value < 0) {
         *buffer++ = '-';
-        value = -value;
+        u = ~u + 1;
     }
 
-    return u64toa(static_cast<uint64_t>(value), buffer);
+    return u64toa(u, buffer);
 }
```
Comment (MOON-CLJ) [inline include/rapidjson/itoa.h]: ref: https://github.com/Tencent/rapidjson/pull/296/commits/e346b933250fd14e9463ae8c542576222973b659

File: include/Parser.h [hunk 2]
```diff
@@ -43,7 +43,7 @@ class PacketParser {
   void processLineResult(err_code_t& err);
 
 
-  std::queue<struct ::iovec> m_requestKeys;
+  std::queue<struct iovec> m_requestKeys;
```
Comment (MOON-CLJ) [inline include/Parser.h]: "struct iovec" "struct ::iovec" both exists in source code，leave "struct iovec"。

File: src/Parser.cpp [hunk 3]
```diff
@@ -133,23 +130,23 @@ void PacketParser::process_packets(err_code_t& err) {
...
           mt_kvPtr->bytes = static_cast<uint32_t>(bytes);
-          mt_kvPtr->bytesRemain = mt_kvPtr->bytes + 1; // 1 for '\n
-          SKIP_BYTES(1);  // " " or "\r"
-          m_state = FSM_GET_BYTES_CAS;
+          mt_kvPtr->bytesRemain = mt_kvPtr->bytes + 1; // 1 for '\n'
+          SKIP_BYTES(1);  // ' ' or '\r'
+          m_state = FSM_GET_BYTES;
         }
         break;
-      case FSM_GET_BYTES_CAS: // got "bytes\r" or "bytes cas\r"
```
Comment (MOON-CLJ) [inline src/Parser.cpp]: state FSM_GET_BYTES_CAS means got cas，but here it's not。so split FSM_GET_BYTES_CAS to FSM_GET_BYTES and  FSM_GET_CAS。make state always means done。

**Comment by MOON-CLJ** [comment]
ping @mckelvin i want to know how do you think about this pr? should i split four commit to four prs?

**Comment by MOON-CLJ** [comment]
ping @mckelvin

**Comment by mckelvin** [comment]
LGTM

**Pull request merged**